### DISCUSSION TITLE
无法正确创建快照

### DISCUSSION THREAD

**Discussion by likesrt** [1Panel-dev/1Panel#5060] [Q&A]
### 联系方式

yuyan@broue.cn

### 1Panel 版本

专业版：V1.10.5-Its

### 问题描述

创建快照时卡在制作快照文件长达三小时。
使用的腾讯cos cos文件中出现了1panel的测试文件。

### 重现步骤

创建快照

### 期待的正确结果

正确备份

### 附加信息

![Clip_2024-04-20_22-49-41](https://github.com/1Panel-dev/1Panel/assets/31208912/fea9eb91-b432-4604-a932-7b1b92c3cc01)

**Comment by wanghe-fit2cloud** [comment]
进入 `日志审计` 菜单，查看一下系统日志。

**Comment by likesrt** [comment]
> 进入 `日志审计` 菜单，查看一下系统日志。

没有在最近日志中查看到相关信息。我打包了最近日志您可以查看一下
https://cdnimg.s3.bitiful.net/uploads/2024/04/20/hHNiMY.zip

**Comment by ssongliu** [comment]
看起来是快照进程断了，重启 1panel 服务后重试一下
